services/datapipeline/src/main/resources/codegen-resources/customization.config

```diff
@@ -1,9 +1,4 @@
 {
-    "simpleMethods" : {
-        "ListPipelines" : {
-            "methodForms" : [[ ]]
-        }
-    },
     "authPolicyActions" : {
```

Comment:
Do we event need auth policy actions anymore either?

Reply:
Didn't know about this one. They aren't hooked up currently. Could be a useful feature to have eventually but not sure it needs to be limited per service if so. I'll probably remove in my follow up CR for enabling the customization I'm working on.